smarts/core/bubble_manager.py

```diff
@@ -618,7 +620,7 @@ def _make_social_agent_id(vehicle_id):
 
     @staticmethod
     def _make_boid_social_agent_id(social_agent_actor):
-        return f"BUBBLE-AGENT-{truncate(social_agent_actor.id, 48)}"
+        return f"BUBBLE-AGENT-{truncate(social_agent_actor.name, 48)}"
```

Comment:
What is the reason for changing this? I am concerned it may have a major affect on the agent ids.

Reply:
`SocialAgentActor` from smarts/sstudio/types does not have an `id` attribute, only `name`

Reply:
Weird, does this ever even get hit then?

Reply:
I am not sure